OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -20,6 +20,14 @@ protected virtual bool ShouldFlee(Squad owner)
 		{
 			return base.ShouldFlee(owner, enemies => !AttackOrFleeFuzzy.Default.CanAttack(owner.Units, enemies));
 		}
+
+		protected Actor FindClosestEnemy(Squad owner)
+		{
+			if (!owner.IsValid)
+				return null;
+
+			return owner.Bot.FindClosestEnemy(owner.Units.FirstOrDefault().CenterPosition);
```

Comment:
We'll need to be careful that this doesn't make units run back/forward if it oscillates between targets.